gcloud/bigquery/table.py

```diff
@@ -761,8 +762,9 @@ def _bool_from_json(value, field):
 
 def _datetime_from_json(value, field):
     if _not_null(value, field):
-        # Field value will be in milliseconds.
-        return _datetime_from_microseconds(1000.0 * float(value))
+        # value will be a float in seconds, to microsecond precision, in UTC.
+        stamp = datetime.datetime.utcfromtimestamp(float(value))
```

Comment:
Can you put this in `gcloud._helpers`? The point of #1051 was to re-use / be consistent on this.

You could also just use `_datetime_from_microseconds(1e6 * float(value))`